easyrsa3/easyrsa

```diff
@@ -100,8 +101,16 @@ cmd_help() {
 
       This mode uses the <filename_base> as the X509 CN."
 			opts="
-        nopass  - do not encrypt the private key (default is encrypted)
-        inline  - create an inline credentials file for this node" ;;
+        nopass  - do not encrypt the private key (default is encrypted)" ;;
+		build-inline) text="
+  build-inline <filename_base>
+      Create an inline file for node <filename_base>
+
+      The inline file is:
+      * created in your PKI directory: ./inline/<filename_base>.conf
+      * comprised of: short node description, node cert, node key, CA cert
+
+      Note: Other inline files must be added manually. eg: ta.key" ;;
```

Comment:
Other inline files should be added automatically if we're going to support it at all, this can be done with a seed/started config and just append the ca, cert, and key here.

Reply:
Other **key** files are outside of easyrsa3's remit and I will not add them until easyrsa3 has code to build them.